modules/dxp/apps/oauth/oauth-api/src/main/java/com/liferay/oauth/util/V10aOAuth.java

```diff
@@ -19,8 +19,9 @@
 import com.liferay.oauth.model.OAuthUser;
 import com.liferay.oauth.service.OAuthApplicationLocalServiceUtil;
 import com.liferay.oauth.service.OAuthUserLocalServiceUtil;
+import com.liferay.osgi.util.ServiceTrackerFactory;
```

Comment:
@dantewang @yuhai ,
If I remembered right, we have some private modules using these MultiVMUtil/SingleVMUtil too.

Also, we are trying to stop people using service tracker in static class in module, this is kind of a hidden dependency which is hard to track if there is anything wrong, so the solution can not be accepted. 

Please hold on this and revisit it after we finished the easier ones.

Thanks.
Tina.